### PR TITLE
Adding Atoti CE Dockerfile

### PR DESCRIPTION
**Pull request by andrewyang94**
This PR adds a Dockerfile that developers can use to quickly install and access the notebooks in the Atoti repo. To build and test, you must have Docker installed and execute the following (at the root of the repo):

```
docker build -f Dockerfile -t atoti-tutorial .
docker run -p 8888:8888 -p 9092:9092 -it atoti-tutorial
```

Then go to the URL of the JupyterLab instance port forwarded from within the container.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Set explicit platform for ARM-based Macbooks
+FROM --platform=linux/amd64 python:3.10.13-slim-bookworm
+
+# Install dependencies in container
+RUN apt-get update && \
+    apt-get install -y nodejs graphviz && \
```
Comment (tibdex) [inline Dockerfile]: You don't need these two lines.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    pip install atoti \
+                atoti-jupyterlab \
+                atoti-aws \
+                atoti-sql
```
Comment (tibdex) [inline Dockerfile]: A little nicer with the "extras" syntax:

```suggestion
    pip install "atoti[aws,jupyterlab,sql]"
```

File: Dockerfile (hunk 1 above)
Comment (andrewyang94) [reply Dockerfile]: Removed! These were old lines that I just cleaned up!

File: Dockerfile (hunk 2 above)
Comment (andrewyang94) [reply Dockerfile]: No need since we're promoting poetry installation method anyways!

**Review by printhellohetal — APPROVED** [review]
Let's do this!

**Pull request merged**